src/gui/activitywidget.h

```diff
@@ -87,10 +91,17 @@ private slots:
     QPushButton *_copyBtn;
 
     QSet<QString> _accountsWithoutActivities;
-    QMap<int, NotificationWidget*> _widgetForNotifId;
+    QMap<Activity::Identifier, NotificationWidget*> _widgetForNotifId;
     QElapsedTimer _guiLogTimer;
     QSet<int> _guiLoggedNotifications;
-    int _notificationRequests;
+    ActivityList _blacklistedActivities;
```

Comment:
We don't persist this information - does it mean users have to acknowledge the server notification every time the client is restarted?

I assume that for notifications with actions, the server will keep track of them and that's how we avoid presenting them to the user multiple times...